DESCRIPTION

```diff
@@ -28,4 +28,4 @@ Imports:
 Suggests:
   optparse
```

Comment:
need to add `rentrez` -- I think to Suggests? There are other functionalities in this package besides the publication script that don't require rentrez (I don't know if they are ever used, but they exist).